external/radiation/python/radiation_driver.py

```diff
@@ -423,6 +421,18 @@ def _GFS_radiation_driver(
 
         scmpsw = dict()
         Diag = dict()
+        Radtend = dict()
```

Comment:
No need to change this, but usually to initialize a dict one can do this: `Radtend = {}`